src/agent/debuglet.ts

```diff
@@ -255,6 +256,20 @@ export class Debuglet extends EventEmitter {
     this.debuggeeRegistered = new CachedPromise();
   }
 
+  //                     0        1        2       3       4        5
+  static LEVELNAMES = [ 'fatal', 'error', 'warn', 'info', 'debug', 'trace'];
+  static logLevelToName(level: number): string {
+    if (typeof level === 'string') {
+      level = parseInt(level, 10);
+    }
+    if (typeof level !== 'number') {
+      level = defaultConfig.logLevel;
+    }
+    if (level < 0) level = 0;
```

Comment:
For clarity, I think we should still use `{ ... }` around the body of the if statements.

Reply:
Style guide is okay with this, and I like the improved readability from brevity.

Reply:
I prefer always using brackets, but if the style guide is ok with this then so am I.